Resources/Prototypes/Recipes/Lathes/Packs/engineering.yml

```diff
@@ -93,3 +93,4 @@
   - HolopadMachineCircuitboard
   - PowerComputerCircuitBoard
   - AtmosMonitoringComputerCircuitBoard
+  - WallmountGeneratorElectronics # L5
```

Comment:
Even though it's tier three and not at all easy, I don't think these should be obtainable. It's already far too easy to get eight bajillion RTGs. I just find it extremely boring, at least when I'm engi, to have these used. (I actually feel the same with about non-damaged RTGs since they're just so plentiful, but it's not super easy to get a actually forever-TEG set up without a better supply of plasma. I am tempted to look into trying that though...)

Reply:
Nah, after giving this some thought, I agree. I'll comment out for now.